docs/file-based-configuration.md

```diff
@@ -0,0 +1,64 @@
+# File-based Configuration
+
+> **Status:** [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md)
+> For more information about the OpenTelemetry configuration specification, see:
+> **[File-based configuration documentation](https://opentelemetry.io/docs/specs/otel/configuration/sdk/)**  
+
+You can configure OpenTelemetry using a YAML file. This method is disabled
+by default and must be explicitly enabled.
+
+To enable file-based configuration, set the following environment variable:
+
+```bash
+OTEL_EXPERIMENTAL_FILE_BASED_CONFIGURATION_ENABLED=true
+```
+
+By default, the value is false.
+
+You can also specify the configuration file path (default: config.yaml):
+
+```bash
+OTEL_EXPERIMENTAL_CONFIG_FILE=/path/to/config.yaml
+```
+
+In your config file you can use environment variables in the format `${ENVIRONMENT_VARIABLE}`
+instead of a fixed value.
+
+You can also use `${ENVIRONMENT_VARIABLE:-value}`, where `value` is the fallback
+if the environment variable is empty or not set.
+
+## Configuration Examples
+
+### Resource Configuration
```

Comment:
Please define the default value.

Reply:
Adding a code block is fine.

> Wouldn’t it be better to add a link to the file in the specification once it’s ready?

If it is not ready then we cannot do anything about it.